File: ProgrammersGuide/WorkingWithProjects/GetNumberOfPagesInProject/GetNumberOfPagesInProjectExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammersGuide.WorkingWithProjects.GetNumberOfPagesInProject
	<TestClass, TestFixture> _
	Public Class GetNumberOfPagesInProjectExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub GetNumberOfPages()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/GetNumberOfPagesInProject/GetNumberOfPages")
	
	            GetNumberOfPagesExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub GetNumberOfPagesForDifferentViews()
	            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/GetNumberOfPagesInProject/GetNumberOfPagesForDifferentViews")
	
	            GetNumberOfPagesForDifferentViewsExample.Program.Main()
        End Sub

	End Class
End Namespace

File: Examples/ProgrammersGuide/WorkingWithProjects/GetNumberOfPagesInProject/GetNumberOfPagesInProjectExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammersGuide.WorkingWithProjects.GetNumberOfPagesInProject
{
    [TestClass, TestFixture]
    public class GetNumberOfPagesInProjectExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void GetNumberOfPages()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/GetNumberOfPagesInProject/GetNumberOfPages");

            GetNumberOfPagesExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void GetNumberOfPagesForDifferentViews()
        {
            TestHelper.SetDataDir("ProgrammersGuide/WorkingWithProjects/GetNumberOfPagesInProject/GetNumberOfPagesForDifferentViews");

            GetNumberOfPagesForDifferentViewsExample.Program.Main();
        }

    }
}
